plop/collector.py

```diff
@@ -159,10 +159,10 @@ def main():
         sys.stderr.flush()
         sys.exit(1)
 
-    if not os.path.exists('profiles'):
-        os.mkdir('profiles')
-    filename = 'profiles/%s-%s.%s' % (args.target, time.strftime('%Y%m%d-%H%M-%S'),
-                                      extension)
+    filename = os.path.join(
+        'profiles',
+        '%s-%s.%s' % (args.target, time.strftime('%Y%m%d-%H%M-%S'), extension))
+    os.makedirs(os.path.dirname(filename), exist_ok=True)
```

Comment:
`os.makedirs` didn't get the `exist_ok` argument until python 3.

It's surprising that absolute paths behave so differently from relative paths here. The intention was for the output to go into `./profiles/`, so I think the answer is to either use `os.path.basename(args.target)` or encode slashes in `args.target` so it doesn't try to create a deep directory tree.